example/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "appcues-react-native-example",
   "description": "Example app for appcues-react-native",
-  "version": "1.0.1",
+  "version": "1.0.1-beta.1",
```

Comment:
The CircleCI caching is a bit too aggressive and that was causing the build to  fail, so I changed the version to invalidate. I'll look into a better fix.